backend/src/main/java/com/huellapositiva/application/controller/ProposalApiController.java

```diff
@@ -310,6 +311,10 @@ public ListedProposalsDto fetchListedProposals(@PathVariable Integer page, @Path
                             responseCode = "200",
                             description = "Ok, email with proposal sent to reviser."
                     ),
+                    @ApiResponse(
+                            responseCode = "400",
+                            description = "Bad request. The ID is not in review pending."
+                    ),
```

Comment:
400 should only be returned when you send a malformed/invalid payload (HTTP request body).
Use 403, 409, 412. If you want to be consistent with other features return 412 although might not be the most appropriate

Reply:
Ok, I have written responseCode="412", I think it is the most appropriate.